nidirect_news/tests/src/Nightwatch/Tests/featuredContentListViewSwitch.js

```diff
@@ -0,0 +1,37 @@
+let newsContentTitles = [];
+let homepageFeaturedTitles = [];
+
+module.exports = {
+  '@tags': ['regression'],
+
+  before: browser => {
+    // Gather the titles of the current 'latest/featured news' content.
+    browser.drupalRelativeURL('/news');
+    browser.elements('css selector', '.card .card__title', function (result) {
```

Comment:
I cannot tell you how long this took to piece together. Between finding a way to handle multiple values from the CSS selector to storing them in an accessible variable for use later on... mind blown.